config.js

```diff
@@ -238,114 +238,130 @@ exports.updateFromArgv = function() {
         }
     }
 
-    if (argv.komiunranked) {
-        if (argv.komiunranked.includes(`auto`)) {
-            console.log("Warning: /--komiunranked auto/ has been renamed to /--komiunranked automatic/\n");
-            // we skip a line here, not below, because this argv may be undefined if not used by bot admin
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.minmaintime, argv.minmaintimeblitz, argv.minmaintimelive, argv.minmaintimecorr, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.minmaintimeranked, argv.minmaintimeblitzranked, argv.minmaintimeliveranked, argv.minmaintimecorrranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.minmaintimeunranked, argv.minmaintimeblitzunranked, argv.minmaintimeliveunranked, argv.minmaintimecorrunranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.maxmaintime, argv.maxmaintimeblitz, argv.maxmaintimelive, argv.maxmaintimecorr, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.maxmaintimeranked, argv.maxmaintimeblitzranked, argv.maxmaintimeliveranked, argv.maxmaintimecorrranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.maxmaintimeunranked, argv.maxmaintimeblitzunranked, argv.maxmaintimeliveunranked, argv.maxmaintimecorrunranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.minperiodtime, argv.minperiodtimeblitz, argv.minperiodtimelive, argv.minperiodtimecorr, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.minperiodtimeranked, argv.minperiodtimeblitzranked, argv.minperiodtimeliveranked, argv.minperiodtimecorrranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.minperiodtimeunranked, argv.minperiodtimeblitzunranked, argv.minperiodtimeliveunranked, argv.minperiodtimecorrunranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.maxperiodtime, argv.maxperiodtimeblitz, argv.maxperiodtimelive, argv.maxperiodtimecorr, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.maxperiodtimeranked, argv.maxperiodtimeblitzranked, argv.maxperiodtimeliveranked, argv.maxperiodtimecorrranked, this.indicatorConsoleDepreciation);
+    depreciatedArgumentIsUsedNewArgumentToUseInsteadUpToThree(argv.maxperiodtimeunranked, argv.maxperiodtimeblitzunranked, argv.maxperiodtimeliveunranked, argv.maxperiodtimecorrunranked, this.indicatorConsoleDepreciation);
+
+    function isArgumentArrayEqualToAuto(argumentToTest, indicatorDepreciation) {
```

Comment:
I would just call this `checkForDeprecatedKomi()` - this is a special purpose function and it's best to name it for what is the task that this function is supposed to accomplish. Especially as it also checks for `null`.

Reply:
in the future we may use this for `notification.user.color` or `notification.autohandicap` which can also have "automatic" as a value

Reply:
Makes sense!